Library/Formula/rakudo-star.rb

```diff
@@ -28,15 +21,35 @@ def install
     ENV.prepend "CPPFLAGS", "-I#{libffi.lib}/libffi-#{libffi.version}/include"
 
     ENV.j1  # An intermittent race condition causes random build failures.
+
+    backends = ["moar"]
+    generate = ["--gen-moar"]
+
     if build.with? "jvm"
-      system "perl", "Configure.pl", "--prefix=#{prefix}", "--backends=parrot,jvm", "--gen-parrot"
-    else
-      system "perl", "Configure.pl", "--prefix=#{prefix}", "--backends=parrot", "--gen-parrot"
+      backends << "jvm"
     end
+    if build.with? "parrot"
+      backends << "parrot"
+      generate << "--gen-parrot"
+    end
+    system "perl", "Configure.pl", "--prefix=#{prefix}", "--backends=" + backends.join(","), *generate
     system "make"
     system "make install"
-    # move the man pages out of the top level into share.
-    mv "#{prefix}/man", share
+
+    # TEMPORARY Workaround for http://stackoverflow.com/questions/9988125/shebang-pointing-to-script-also-having-shebang-is-effectively-ignored.
+    # rakudo-star 2014.12.1 has shebang lines in some scripts which point to other scripts with shebang
+    # lines, which will be ignored/fail on MacOS. This uses /usr/bin/env instead which is also implemented
+    # upstream (not released yet).
+    rakudo_shebangs = `grep --recursive --files-with-matches ^#!#{bin} #{prefix}`
+    rakudo_shebang_files = rakudo_shebangs.lines.sort.uniq
+    rakudo_shebang_files.map! {|f| Pathname(f.chomp)}
+    inreplace rakudo_shebang_files, %r{^(#!#{bin}/)}, "#!/usr/bin/env "
+
+    # Move the man pages out of the top level into share.
```

Comment:
I think this might have been added because of this homebrew requirement: "Homebrew expects to find man pages in [prefix]/share/man/..., and not in [prefix]/man/....".
Rakudo star installs to man, and thus these lines move it to share/man where homebrew would expect it. Please correct me if it should be done otherwise. There is no configure switch available for it.
I expanded the comment to explain that only some of the optional backends (parrot) do install man pages, so the directory might be missing.